temporalio/lib/temporalio/converters/payload_converter/composite.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 require 'temporalio/api'
+require 'temporalio/error'
```

Comment:
The requires have to be in alphabetical order, can you move this to the bottom of the list?